src/main/java/mate/academy/spring/validation/Email.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Email {
```

Comment:
Damn, I probably should've done the same :)

Reply:
:harold: